src/card.css.ts

```diff
@@ -177,6 +177,22 @@ export const mediaCardContent = style([
     },
 ]);
 
+export const mediaCardAsset = style([
+    {
+        position: 'absolute',
+        zIndex: 1,
+    },
+    {
+        '@media': {
+            [mq.desktopOrBigger]: {
+                paddingLeft: 24,
+                paddingTop: 24,
+                paddingBottom: 32,
```

Comment:
@aweell, it's ok to have different paddings in desktop? I don't see this in the specs.

If these are needed, we should move them to the Box element instead of creating a new media query here @thiagoaraujo-tlf:

```ts
<Box paddingLeft={{mobile: 16, desktop: 24}} paddingTop={{mobile: 16, desktop: 24}}>
```
Also, I think `paddingRight` and `paddingBottom` are not needed, just left and top

Reply:
Yes, the asset has a padding left and top of 16 in mobile and 24 in desktop.

Mobile:
![Screenshot 2023-07-26 at 17 02 43](https://github.com/Telefonica/mistica-web/assets/44420072/f082cd8f-3777-4151-87c8-ff872c117646)

Desktop:
![Screenshot 2023-07-26 at 17 02 46](https://github.com/Telefonica/mistica-web/assets/44420072/f7e55a0a-3c4b-4cf4-b2c2-cbf5229a6f20)

This is intended to align the asset to the content since the padding of the content increases in desktop.

The other changes, it seems they're related the content padding? They are already working in the current component.

Mobile:
![Screenshot 2023-07-26 at 17 06 17](https://github.com/Telefonica/mistica-web/assets/44420072/bcdbe8ea-2b94-4c16-8a7d-8a0fb6cbefa7)

Desktop:
![Screenshot 2023-07-26 at 17 05 19](https://github.com/Telefonica/mistica-web/assets/44420072/775758bb-7b6f-4945-8a01-8c59dce5c319)

Reply:
I just added icon padding for when it's on desktop @atabel